profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.info

```diff
@@ -4,5 +4,12 @@ package = multisite drupal toolbox
 core = 7.x
 
 dependencies[] = libraries
+dependencies[] = multisite_config
+dependencies[] = linkchecker
+dependencies[] = entity
```

Comment:
multisite_config already has a dependency to entity so we should probably remove it.